example5/index.html

```diff
@@ -31,10 +31,13 @@
         <script type="importmap">
             {
                 "imports": {
-                "three": "https://cdn.jsdelivr.net/npm/three@0.137.5/build/three.module.js",
                 "three/examples/jsm/controls/OrbitControls":"./jsm/controls/OrbitControls.js",
                 "three/examples/jsm/libs/stats.module":"./jsm/libs/stats.module.js"
```

Comment:
I have noticed that in examples 5 and 6 there are already some imports, but haven't seen them being actually used, can we remove those?